x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_settings_collector.js

```diff
@@ -8,23 +8,48 @@ import { get } from 'lodash';
 import { XPACK_DEFAULT_ADMIN_EMAIL_UI_SETTING } from '../../../../../server/lib/constants';
 import { KIBANA_SETTINGS_TYPE } from '../../../common/constants';
 import { getKibanaInfoForStats } from '../lib';
+import { CLUSTER_ALERTS_ADDRESS_CONFIG_KEY } from '../../lib/constants';
+
+let loggedDeprecationWarning = false;
+
+export function resetDeprecationWarning() {
```

Comment:
This only exists to enable test cases to reset the module state. I realize how ugly this is, but I was weighing the tradeoff of this vs a larger, more complex solution. I chose this because it is a short-lived feature that we will remove with 7.0